### DISCUSSION TITLE
Agent Ignorig  data - CPU and Network

### DISCUSSION THREAD

**Discussion by SanthoshKumar18245** [grafana-cold-storage/agent#6228] [General]
i am configuring the grafana agent to scrape the metrics which is prometheus node_exporter can do... but i am getting some Errors

my config file,,,

server:
  log_level: debug

metrics:
  wal_directory: /tmp/agent
  global:
    scrape_interval: 60s
    remote_write:
      - url: http://127.0.0.1:9009/api/v1/push
        headers:
          X-Scope-OrgID: monitoring
integrations:
  node_exporter:
    enabled: true
    rootfs_path: /root
    sysfs_path: /sys
    procfs_path: /proc
    udev_data_path: /run/udev/data
    enable_collectors:
      - cpu
      - diskstats
      - filesystem
      - loadavg
      - netdev

<img width="936" alt="image" src="https://github.com/grafana/agent/assets/133011171/686a9a18-fe04-4409-89f4-190a7b2d8320">
@bcotton  please discuss with me

**Comment by bingkunyangvungle** [comment]
Same. Anyone can help?
